docs/reference/ingest/search-ingest-pipelines.asciidoc

```diff
@@ -0,0 +1,313 @@
+===== `<index-name>@custom` Reference
+
+This pipeline is empty to start (no processors), but can be added to via the Kibana UI either through the Pipelines
+tab of your index, or from the *Stack Management > Ingest Pipelines* page.
+Unlike the `ent-search-generic-ingestion` pipeline and the `<index-name>` pipeline, this pipeline is NOT "managed".
+
+You are encouraged to make additions and edits to this pipeline, provided its name remains the same.
+This provides users a convenient hook from which to add their own custom processing and transformations for their *Search* data.
+Be sure to read the <<ingest, docs for ingest pipelines>> so that you know what all is available to you here.
+
+[WARNING]
+=========================
+You should not rename this pipeline.
+=========================
+
+[discrete#ingest-pipeline-search-upgrading-notes]
+=== Upgrading notes
+
+.Expand to see upgrading notes
+[%collapsible%closed]
+=============
+
+* `app_search_crawler` - Since 8.3, {app-search-crawler} has utilized this pipeline to power its binary content
+extraction.
+  You can read more about this pipeline and its usage in the {app-search-ref}/web-crawler-reference.html#web-crawler-reference-binary-content-extraction[App Search Guide].
+  When upgrading from 8.3 to 8.5+, be sure to note any changes that you made to the `app_search_crawler` pipeline.
+  These changes should be re-applied to each index's `<index-name>@custom` pipeline in order to ensure a consistent data processing experience.
+  In 8.5+, the <<ingest-pipeline-search-pipeline-settings, index setting to enable binary content>> is required *in addition* to the configurations mentioned in the {app-search-ref}/web-crawler-reference.html#web-crawler-reference-binary-content-extraction[App Search Guide].
+
+* `ent_search_crawler` - Since 8.4, the Elastic web crawler has utilized this pipeline to power its binary content extraction.
+  You can read more about this pipeline and its usage in the {enterprise-search-ref}/crawler-managing.html#crawler-managing-binary-content[Elastic web crawler Guide].
+  When upgrading from 8.4 to 8.5+, be sure to note any changes that you made to the `ent_search_crawler` pipeline.
+These changes should be re-applied to each index's `<index-name>@custom` pipeline in order to ensure a consistent data processing experience.
+  In 8.5+, the <<ingest-pipeline-search-pipeline-settings, index setting to enable binary content>> is required *in addition* to the configurations mentioned in the {enterprise-search-ref}/crawler-managing.html#crawler-managing-binary-content[Elastic web crawler Guide].
+
+* `ent-search-generic-ingestion` - Since 8.5, Native Connectors, Connector Clients, and new (>8.4) Elastic web crawler indices will all make use of this pipeline by default.
+  A natural evolution from `app_search_crawler` and `ent_search_crawler`, this pipeline is designed to meet more generic needs across *Search*.
```

Comment:
Rather than substituting _Search_ for _Enterprise Search_, we would ideally rewrite sentences to avoid the proper noun. For example, in this sentence I think we could just drop "across Search".

In other cases, we could refer to the Content UI, search use cases, or use other language to re-frame this away from the former Enterprise Search solution. Sorry, I should have caught that before we migrated this content, but I overlooked the use of the variable, which is used almost exclusively in this doc.

Reply:
No worries I needed a second pair of eyes here in any case because I was focused on getting a green build rather than the content per se.